assets/src/components/v2/dup/no_data.tsx

```diff
@@ -8,15 +8,19 @@ import NormalHeader from "./normal_header";
 export const REPLACEMENTS = {
   WTC: "World Trade Center",
   Malden: "Malden Center",
-} as {[key:string]: string};
+} as { [key: string]: string };
 
-const NoData: ComponentType = () => {
+interface Props {
+  include_header?: boolean;
+}
+
+const NoData: ComponentType<Props> = ({ include_header }) => {
   let stationName = useOutfrontStation() || "Transit information";
   stationName = REPLACEMENTS[stationName] || stationName;
 
   return (
     <div className="no-data__container">
-      <NormalHeader text={stationName} />
+      {include_header && <NormalHeader text={stationName} />}
```

Comment:
These changes set up the ability to provide `include_header` to no_data, but all the instances of it in `dup.tsx` are set to true, so were these changes needed?

Reply:
It is there to prevent the `departures_no_data` widget from rendering a duplicate header. The slot for `departures_no_data` is `:main_content_x` which means that a header candidate will still be given to the frontend to render. In the error state you added in `dup.tsx`, that specific no data state cannot render a header without hardcoding one in the component. So having this conditional allows each no data state to render what it needs to.

Reply:
Ohhh I see now. 💯